doc/build-instructions/cmake.md

```diff
@@ -39,6 +39,41 @@ can be followed with the following modifications:
 
 :pencil: For more CMake info see [OMR CMake documentation](https://github.com/eclipse/omr/blob/master/doc/BuildingWithCMake.md)
 
+## Advanced: Building the vm without building openjdk
+
+- Create build directory (its relation to the openj9/omr source trees is not important, however the openj9 directory should not be the same as the build directory)
```

Comment:
Devin, just so I'm clear on the instructions, to build this way do we still need to follow some of the earlier steps listed in the [OpenJ9 build instructions](https://www.eclipse.org/openj9/oj9_build.html)?  That is:

1. Clone the openj9-openjdk-jdk{8|11|14|...} repository, and `cd` to that directory
1. Run `bash ./get_source.sh`
1. Run `bash ./configure --with-freemarker-jar=/root/freemarker.jar --with-cmake`
1. `mkdir build; cd build`
1. Create the `openj9_version_info.h` file described below
1. Invoke `cmake` as described below

If so, I'm not sure that covers the case Andrew @andrewcraik was describing in [this comment in Issue 9195](https://github.com/eclipse/openj9/issues/9195#issuecomment-615252009).  The scenario he was describing involved downloading a previously built JDK, modifying some compiler source files and rebuilding the JIT.

Members of the compiler team often do this to rebuild a JIT while investigating a performance problem or a compiler bug to ensure that they are working with a JDK that is known to exhibit a particular problem or characteristics, while only varying the JIT DLL itself.

Sorry if I've misunderstood your instructions!

Reply:
> If so, I'm not sure that covers the case Andrew @andrewcraik was describing in [this comment in Issue 9195](https://github.com/eclipse/openj9/issues/9195#issuecomment-615252009). The scenario he was describing involved downloading a previously built JDK, modifying some compiler source files and rebuilding the JIT.

We don't ship source code with builds at Adopt nor on any of our Jenkins generated builds at OpenJ9 so this isn't supported today even. If you want to build a change just in the JIT for a particular build you can do so using Devin's instructions by just doing a `java -version` on the target build, extracting the SHAs, and checking out those SHAs before you start Devin's instructions.

Then do `make compiler` (or whatever the target is to build just the JIT) and the output will be your libj9jit29.so which you can then copy over to your target build or just run your target build with `-XXjitdirectory=` to point it to the library you just built.

Reply:
Thanks, Filip @fjeremic.  I thought I just didn't know the right place to look for builds that are packaged with the source code.

Devin, having tried it out now, I see that steps 1. to 3. above are not needed to do this - just steps 4. to 6.  I know the title of the section says, **Advanced: Building the vm without building openjdk**, but it might be good to explicitly state that you don't need to clone the OpenJDK repository or run `configure`.